readme.md

```diff
@@ -31,32 +33,101 @@ Don't see it? Try reinstalling Package Control. Altenatively, add the repository
 
 If it doesn't work, please file an issue.
 
+If you want contribute or use the latest commits simply git clone the repository `https://github.com/mrjoelkemp/Dependents` to your Packages directory with a directory name of `Dependents`. From sublime choose `Preferences -> Browse Packages`  to easily determine the path to the Packages directory for your install.  Be sure to remove the package via package control first if you had already done so to avoid package control updating the package.
+
+### Configuring settings
+
+Rather than using a typical sublime settings file you define settings on a *per-project basis* by creating a `.deprc` file in a project (open folder) root. The paths you will specify are then **relative** to the folder although absolute paths can be used.  Following are a few examples to help you get started.  Below are the current settings that the package recognizes.
+
+Say you open a folder/project in sublime to `/path/to/projects/someproject`.  Put a .deprc file in the project root  `/someproject`.   
+
+For javascript if you want dependents to search all the way to this root (`/someproject`) then put this in the `.deprc` file
+```
+{
+  "root": "./",
+}
+```
+Use `"./"` to signify the root and NOT just a `""`.
+
+But if your javascript code was all contained within a subdirectory of `/someproject` you could limit your dependents search there.
+```
+{
+   "root": "root": "assets/js"
+}
+```
+
+Works the same for sass but use `sass_root`
+```
+{
+  "sass_root": "assets/sass",
+}
+```
+
+
+Here is an example of using some other optional settings.  You can also combine js and sass roots in the same `.deprc`.  See below for all possible settings. At a minimum you will need at last a `root` or `sass_root` in any `.deprc` file
+```
+{
+  "root": "assets/js",
+  "config": "assets/js/config.js",  
+  "exclude": ['jquery.js', 'require.js'],  
+  "build_config": "assets/js/build.json", 
+  "node_path": "/my/node/install/folder", 
+  "sass_root": "assets/sass",  
+}
+```
+
+Tip: If you like to open a separate windows in sublime for subdirectories of your project just put a `.deprc` file there too with the reduced/correct relative paths.   
+
+
+**javascript specific settings**
```

Comment:
I'd put this up above the "say you open..." user story. This way, it serves as a TL;DR for folks who want to try a quick setup. In my experience, no one even wants to read the changelog messages that show up when the package updates.

I still think the user story is valuable, for folks that want more of a walkthrough.